modules/components/tag-input.ts

```diff
@@ -701,6 +707,74 @@ export class TagInputComponent extends TagInputAccessor implements OnInit {
         }
     }
 
+
+    /**
+     * @name DRAG_N_DROP_KEY
+     * @type {string}
+     */
+    private static DRAG_N_DROP_KEY: string = "dragged tag";
```

Comment:
But this constant is only for using inside this class, Are you sure that the global space is better place for this constant?